modules/script/script-impl/src/test/resources/myapp/script.js

```diff
@@ -0,0 +1,6 @@
+exports.execute = function() {
+    let bean = __.newBean('com.enonic.xp.script.graaljs.impl.ComparingApproaches');
+    bean.setValue("Source Name 1");
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/enonic/xp/pullRequest?prid=8692791)
```suggestion
    bean.setValue('Source Name 1');
```